src/raw/mod.rs

```diff
@@ -794,84 +920,30 @@ impl<'f, A: Automaton> Stream<'f, A> {
         }
     }
 
-    /// Convert this stream into a vector of byte strings and outputs.
-    ///
-    /// Note that this creates a new allocation for every key in the stream.
-    pub fn into_byte_vec(mut self) -> Vec<(Vec<u8>, u64)> {
-        let mut vs = vec![];
-        while let Some((k, v)) = self.next() {
-            vs.push((k.to_vec(), v.value()));
-        }
-        vs
-    }
```

Comment:
Why were all of these methods removed?

Reply:
It wasn't removed, it is because the git doesn't understand that it is a new struct named `StreamWithState` that I created and didn't implement those methods on it to keep the code easier to understand.